reconcile/utils/internal_groups/models.py

```diff
@@ -21,6 +23,9 @@ def __eq__(self, other: object) -> bool:
             raise NotImplementedError("Cannot compare to non Entity objects.")
         return self.type == other.type and self.id == other.id
 
+    def __lt__(self, other: Entity) -> bool:
+        return self.id < other.id
```

Comment:
@chassing If we use `frozen = True` as `Config` like https://docs.pydantic.dev/1.10/usage/model_config/, then we can save `__eq__` and `__lt__`, let it auto generate `__hash__`. So we can directly compare them like `set(self.members) == set(other.members)`

Reply:
Thank you. Please take a look at https://github.com/app-sre/qontract-reconcile/pull/3749